content/docs/reference/cli/pulumi_new.md

```diff
@@ -14,12 +14,12 @@ To create a project from a specific template, pass the template name (such as `a
 or `azure-python`).  If no template name is provided, a list of suggested templates will be presented
 which can be selected interactively.
 
-By default, a stack created using the pulumi.com backend will use the pulumi.com secrets
+By default, a stack created using the Pulumi Service backend will use the Pulumi Service secrets
```

Comment:
I think the CLI descriptions have to be updated upstream at https://github.com/pulumi/pulumi/blob/master/cmd/new.go for example? Not sure if that has been done.

Reply:
That’s right, yeah — I bypassed the generated docs for this, but I’m happy to have a look at that next!